lib/Networkteam/Import/DataProvider/CsvDataProvider.php

```diff
@@ -27,7 +31,9 @@ class CsvDataProvider implements DataProviderInterface
     protected $options = array(
         self::KEY_DELIMITER => ',',
         self::KEY_ENCLOSURE => '"',
-        self::KEY_USE_HEADER_ROW => true
+        self::KEY_USE_HEADER_ROW => true,
+        self::KEY_FILE_ENCODING => 'UTF-8',
```

Comment:
Excuse my interruption but I don't think it's a good idea to handle validation of file here in the provider ...

Reply:
Yes ;)

I discussed this with Kai and we'll restructure the Importer a bit, so it takes a `DataProvider` in the `import()` method. This way we split the creation (and validation) of a data provider from the actual run of the importer. The current design has some issues if the importer is a singleton, since it keeps state between runs.

This will need a new major version since the API changes.